GameData/RP-0/changelog.cfg

```
@@ -24,13 +24,14 @@ KERBALCHANGELOG
 			subchange = should move tag
 		}
 	}
+	
 	VERSION
```

Auto-generated potential formatting fix:
```suggestion
	}

	VERSION
```